lib/status/status.go

```diff
@@ -557,3 +572,91 @@ const (
 	// roleTag is the name of the tag containing node role
 	roleTag = "role"
 )
+
+// FromAlertManager collects alerts from the prometheus alertmanager deployed to the cluster
+func FromAlertManager(ctx context.Context, dnsAddr string) ([]*models.GettableAlert, error) {
+	client, err := httplib.GetPlanetClient(httplib.WithLocalResolver(dnsAddr))
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	transport := httptransport.NewWithClient(defaults.AlertmanagerServiceAddr, "/api/v2",
+		[]string{"http"}, client)
+
+	am := alertmanager.New(transport, nil)
+	getOk, err := am.Alert.GetAlerts(alert.NewGetAlertsParams().WithContext(ctx))
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	return filterAlerts(getOk.Payload), nil
+}
+
+const (
+	alertname    = "alertname"
```

Comment:
These constants here and the method below seem to be Alertmanager-specific, how about moving them to a dedicated module (file) in this package (`alertmanager.go`)?